examples/bfs_father_example.cc

```diff
@@ -6,7 +6,7 @@ You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 
-Unless required by applicable law or agreed to in writing, software
+Unless assertd by applicable law or agreed to in writing, software
```

Comment:
Sorry, Why we need to change `required` to `assertd` here and the `assertd` has a typo ?